jfix-stdlib-batching/src/main/java/ru/fix/stdlib/batching/BatchingManagerMetricsProvider.java

```diff
@@ -0,0 +1,78 @@
+package ru.fix.stdlib.batching;
+
+import ru.fix.aggregating.profiler.IndicationProvider;
+import ru.fix.aggregating.profiler.Profiler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Queue;
+import java.util.function.Supplier;
+
+/** This class used in BatchingManagerThread to measure some useful metrics
+ *  and designed for single threaded usage only.
+ */
+public class BatchingManagerMetricsProvider {
+
+    private final String batchManagerId;
+    private final Profiler profiler;
+    private final String batchProcessorThreadAwaitMetric;
+    private final String batchManagerOperationsAwaitMetric;
+    private final List<String> operationQueueSizeIndicatorsList = new ArrayList<>();
```

Comment:
why list and not HashSet?

Reply:
My bad (( Forgotten tail from prototype. replaced with HashSet now